gc/base/standard/Scavenger.cpp

```diff
@@ -348,6 +348,13 @@ MM_Scavenger::collectorShutdown(MM_GCExtensionsBase* extensions)
 {
 #if defined(OMR_GC_CONCURRENT_SCAVENGER)
 	if (IS_CONCURRENT_ENABLED) {
+		/*
+		 * Normally to stop CS Cycle we need to call completeConcurrentCycle(env);
+		 * In this path however we can assume there is no mutator thrads activity.
+		 * So, we can just change state at this point.
+		 */
```

Comment:
There is a typo in the commit message, PR description and code. Also, first person should be avoided. Suggesting a rephrase:
```suggestion
		/*
		 * completeConcurrentCycle(env) is normally invoked to stop a CS cycle.
		 * However, it can be assumed that there is no activity of mutator threads
		 * in the shutdown path. So, a change to the idle state is sufficient at
		 * this point.
		 */
```

Reply:
fixed